apps/server/lib/http/auth.ts

```diff
@@ -0,0 +1,70 @@
+import {
+	JellyfishKernel,
+	SessionContract,
+} from '@balena/jellyfish-types/build/core';
+import bcrypt from 'bcrypt';
+import _ from 'lodash';
+
+// TODO: Move this definition elsewhere, and inject it into the middleware.
+const validateSessionIdAndTokenFunc =
```

Comment:
We normally don't name functions with `Func` etc in the name. I've done similar things in other languages before, but TypeScript makes this unnecessary. :+1:

Reply:
My goal here was to avoid confusion, since I use the higher-order function to create a validator which is later assigned to a const with a similar name. Anyways, the `Func` suffix was removed, and the functions renamed.